ingest.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
I think this is more flexible, you can do the same in the other file too.

Reply:
This is not done in the other two scripts (`start` and `install`).